components/push/ffi/Cargo.toml

```diff
@@ -31,6 +31,10 @@ features = ["sqlcipher", "limits", "functions"]
 [dependencies.sync15]
 path = "../../sync15"
 
+[features]
+rust-http-stack = ["viaduct/rust-http-stack"]
```

Comment:
The megazords turn off this feature (or they attempt to -- there have been some indications in `cargo` issues that this behavior might change to make builds faster, but even if they do, we should still use the FFI version since we explicitly turn it on in LibViaduct.kt).